_notices/rsn0029.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 29 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Dropping Python v3.8 in v23.06"
+notice_author: RAPIDS TPM
+notice_status: In Progress
+notice_status_color: yellow
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v23.06+"
+notice_created: 2023-04-20
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2023-04-20
+---
+
+## Overview
+
+Python 3.8 support will be dropped in our upcoming release of v23.06 scheduled for June 2023. We are picking up Python 3.9 support at same time, leaving us supporting Python 3.9 and 3.10.

```

Comment:
Just a language question -- should the first person plural "we" be used here? There's a tight association between team and product if so. Consider the brand voice and messaging, I am fine with either approach but I know RAPIDS oscillates on this voicing across our docs and website content.

```suggestion
Python 3.8 support will be dropped in the upcoming release of v23.06, scheduled for June 2023. Python 3.9 support will be added at the same time, so that v23.06 will support Python 3.9 and 3.10.
```